src/commands/classified/external.rs

```diff
@@ -304,6 +373,23 @@ async fn run_with_stdin(
                 }
             }
 
+            if is_last {
+                if let Ok(status) = popen.wait() {
```

Comment:
Don't .wait() a popen.  That's a bug we fixed recently: https://github.com/nushell/nushell/pull/1211/files

Reply:
We have two wait strategies here. We wait in one case (when it's last) and we poll when it isn't (where .wait() here was the cause of the bug) I believe.

Reply:
Ahh, okay, got it.

Reply:
Actually, @jonathandturner is unfortunately right, at least through my investigation. `wait` seemed to be a problem no matter what. What happens if you run an interactive command (e.g., `vim`) and resize the terminal? If it works for you, I'll try it out and see what happens.

Reply:
Can confirm.

Sigh.

Reply:
mmm... Got interesting things. See, in some cases this code uses wait(), in other cases polling. It will .wait() if it's an external that's given an `$it`.

I will land this PR and let's continue from there. The point was to make a head start and continue refactoring our way out to unify all things (gif needed)

Reply:
To be more clear. With this PR, whenever an external has the string `"$it"` in it then it will be spawned and `.wait()`'d, if not, whether it's the last command or not, it will be polled.